test/extensions/transport_sockets/tls/BUILD

```diff
@@ -149,28 +140,44 @@ envoy_cc_test_library(
     ],
 )
 
-envoy_cc_test(
-    name = "handshaker_test",
-    srcs = ["handshaker_test.cc"],
-    data = [
-        "gen_unittest_certs.sh",
-        "//test/config/integration/certs",
-        "//test/extensions/transport_sockets/tls/test_data:certs",
-    ],
-    external_deps = ["ssl"],
-    # TODO(sunjayBhatia): Diagnose openssl DLL load issue on Windows
-    # See: https://github.com/envoyproxy/envoy/pull/13276
-    tags = ["flaky_on_windows"],
-    deps = [
-        ":ssl_socket_test",
-        ":ssl_test_utils",
-        "//source/common/stream_info:stream_info_lib",
-        "//source/extensions/transport_sockets/tls:ssl_handshaker_lib",
-        "//test/mocks/buffer:buffer_mocks",
-        "//test/mocks/network:network_mocks",
-        "//test/mocks/runtime:runtime_mocks",
-        "//test/mocks/server:server_mocks",
-        "//test/mocks/ssl:ssl_mocks",
-        "//test/mocks/stats:stats_mocks",
-    ],
-)
+#envoy_cc_test_library(
+#    name = "test_private_key_method_provider_test_lib",
+#    srcs = [
+#        "test_private_key_method_provider.cc",
+#    ],
+#    hdrs = [
+#        "test_private_key_method_provider.h",
+#    ],
+#    external_deps = ["ssl"],
+#    deps = [
+#        "//include/envoy/api:api_interface",
+#        "//include/envoy/event:dispatcher_interface",
+#        "//include/envoy/server:transport_socket_config_interface",
+#        "//include/envoy/ssl/private_key:private_key_config_interface",
+#        "//include/envoy/ssl/private_key:private_key_interface",
+#        "//source/common/config:utility_lib",
+#        "//source/common/protobuf:utility_lib",
+#        "@envoy_api//envoy/extensions/transport_sockets/tls/v3:pkg_cc_proto",
+#    ],
+#)
+
+#envoy_cc_test(
```

Comment:
What's happening with these tests? Should we comment about why we don't enable these tests?

Reply:
There are tickets open for these: https://issues.redhat.com/browse/MAISTRA-2120 and https://issues.redhat.com/projects/MAISTRA/issues/MAISTRA-2121. I know this is ugly, but it makes it easier to merge in upstream changes (if marginally).

Reply:
fair enough